addons/udes_stock/models/stock_warehouse.py

```diff
@@ -160,6 +160,6 @@ def _get_a_type(self, picking_type_name):
             lambda pt: pt.name == picking_type_name)
 
         if not put:
-            put = None
+            put = self.int_type_id
```

Comment:
Id like to see some explanation of why this was necessary in the commit. I'm sure we only just changed it to None.